CONFIGURATION.md

```diff
@@ -89,6 +89,9 @@ modules:
   # Probe fails if SSL is not present.
   [ fail_if_not_ssl: <boolean> | default = false ]
 
+  # Probe fails if a defined body_size_limit is exceeded.
+  [ fail_if_body_too_large: <boolean> | default = true ]
```

Comment:
can you also add an example in the example.yml?

Reply:
Done in [081a0e3](https://github.com/prometheus/blackbox_exporter/pull/1276/commits/081a0e33ad59664be888e1644dbd2b7668e4925c)